.config/config.json

```diff
@@ -20,7 +20,7 @@
   "staticUrls": {
     "dev": "dev-biohubbc.apps.silver.devops.gov.bc.ca",
     "test": "test-biohubbc.apps.silver.devops.gov.bc.ca",
-    "prod": "biohubbc.apps.silver.devops.gov.bc.ca"
+    "prod": "sims.nrs.gov.bc.ca"
```

Comment:
We'll need to specifically test this change in OpenShift. 

It might be fine, but it might also cause trouble if the pods need the internal openshift route to communicate properly. If that ends up being the case, we could add a 4th field here called something like "prod-vanity-url", and then it can pass the sims URL to the appHost env var, and the rest of them can use the original internal url.